src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -45,6 +49,12 @@ public void start(final Map<String, String> props) {
     config = new JdbcSinkConfig(props);
     initWriter();
     remainingRetries = config.maxRetries;
+    try {
+      reporter = context.errantRecordReporter(); // may be null if DLQ not enabled
+    } catch (NoClassDefFoundError e) {
```

Comment:
Have you tested that `NoClassDefFoundError` is the exception to catch? Based on my experience on oracle cdc, the actual exception thrown can be unexpected.

Reply:
Also, jenkins test is red

Reply:
Yes, there's actually two, but I can't add that one until we want to merge or it doesn't let me run tests due to IDE saying that the error will never be thrown (which is wrong). You can see here in https://cwiki.apache.org/confluence/display/KAFKA/KIP-610%3A+Error+Reporting+in+Sink+Connectors which errors should be accounted for.